analysis/tests/cross_partition_combiners_test.py

```diff
@@ -71,5 +72,46 @@ def test_metric_utility_empty_partition(self):
         self.assertEqual(output.num_empty_partitions, 1)
 
 
+# Dataclasses for DataclassHelpersTests
+@dataclasses.dataclass
+class dataclass1:
```

Comment:
nit: I think more descriptive names would be helpful. E.g. `flatdataclass` and `nesteddataclass` perhaps

Reply:
good point, I named `OuterClass` and `InnerClass`